ModelBundle/Repository/RedirectionRepository.php

```diff
@@ -12,4 +12,21 @@
 class RedirectionRepository extends AbstractAggregateRepository implements RedirectionRepositoryInterface
 {
     use PaginationTrait;
+
+    /**
+     * @param string $nodeId
+     * @param string $language
+     *
+     * @return array
+     */
+    public function findByNode($nodeId, $language){
+        $qa = $this->createAggregationQuery();
+
+        $qa->match(array(
+            'nodeId' => $nodeId,
+            'locale' => $language,
```

Comment:
There's a difference between language and locale. If the param is a locale, you should call it so instead of language